e2e/test/scenarios/dashboard-filters/dashboard-filters-reset-clear.cy.spec.ts

```diff
@@ -566,249 +603,505 @@ describe("scenarios > dashboard > filters > reset & clear", () => {
-    checkStatusIcon(DEFAULT_NON_REQUIRED, "reset");
-    resetButton(DEFAULT_NON_REQUIRED).click();
-    filter(DEFAULT_NON_REQUIRED).should("have.text", defaultValueFormatted);
-    checkStatusIcon(DEFAULT_NON_REQUIRED, "clear");
-
-    cy.log("has default value, required, value same as default");
-    checkStatusIcon(DEFAULT_REQUIRED, "none");
-
-    cy.log("has default value, required, current value different than default");
-    updateValue(DEFAULT_REQUIRED, otherValue);
-    filter(DEFAULT_REQUIRED).should("have.text", otherValueFormatted);
-    checkStatusIcon(DEFAULT_REQUIRED, "reset");
-    resetButton(DEFAULT_REQUIRED).click();
-    filter(DEFAULT_REQUIRED).should("have.text", defaultValueFormatted);
-    checkStatusIcon(DEFAULT_REQUIRED, "none");
-
-    checkParameterSidebarDefaultValue({
-      defaultValueFormatted,
-      otherValue,
-      otherValueFormatted,
-      setValue,
-      updateValue,
+    it("works across all tabs with 'auto-apply filters' off", () => {
+      createDashboardWithParameterInEachTab({
+        autoApplyFilters: false,
+        parameters: [PARAMETER_A_DEFAULT_VALUE, PARAMETER_B_DEFAULT_VALUE],
+      });
+      checkResetAllFiltersToDefaultWorksAcrossTabs({ autoApplyFilters: false });
     });
-  }
+  });
+});
 
-  function checkParameterSidebarDefaultValue<T = string>({
+function createDashboardWithParameters(
+  questionDetails: StructuredQuestionDetails,
+  targetField: LocalFieldReference,
+  parameters: DashboardDetails["parameters"],
+) {
+  createQuestionAndDashboard({
+    questionDetails,
+    dashboardDetails: {
+      parameters,
+    },
+  }).then(({ body: { dashboard_id, card_id } }) => {
+    updateDashboardCards({
+      dashboard_id,
+      cards: [
+        {
+          parameter_mappings: parameters?.map(parameter => ({
+            parameter_id: parameter.id,
+            card_id: checkNotNull(card_id),
+            target: ["dimension", targetField],
+          })),
+        },
+      ],
+    });
+
+    visitDashboard(dashboard_id);
+  });
+}
+
+function checkStatusIcon(
+  label: string,
+  /**
+   * Use 'none' when no icon should be visible.
+   */
+  icon: "chevron" | "reset" | "clear" | "none",
+) {
+  clearIcon(label).should(icon === "clear" ? "be.visible" : "not.exist");
+  resetIcon(label).should(icon === "reset" ? "be.visible" : "not.exist");
+  chevronIcon(label).should(icon === "chevron" ? "be.visible" : "not.exist");
+}
+
+function checkDashboardParameters<T = string>({
+  defaultValueFormatted,
+  otherValue,
+  otherValueFormatted,
+  setValue,
+  updateValue = setValue,
+}: {
+  defaultValueFormatted: string;
+  otherValue: T;
+  otherValueFormatted: string;
+  setValue: (label: string, value: T) => void;
+  updateValue?: (label: string, value: T) => void;
+}) {
+  cy.log("no default value, non-required, no current value");
+  checkStatusIcon(NO_DEFAULT_NON_REQUIRED, "chevron");
+  checkResetAllFiltersHidden();
```

Comment:
FYI this function (`checkDashboardParameters`) runs for every dashboard parameter scenario from #46050